v1/proto/service/gribi.proto

```diff
@@ -52,8 +52,12 @@ service gRIBI {
 message ModifyRequest {
   // A group of requests to add/modify/remove a single AFT entry.
   // gRIBI server relies on clients being conservative in what they send.
-  // As such, gRIBI server is expected to process the AFTOperations per the
-  // received order.
+  // As such, when processing AFTOperations, a gRIBI server is expected to:
+  //  * process AFTOperations per the received order.
+  //  * try not to close the RPC session due to errors in AFTOperation
+  //    processing (e.g. invalid network instance, unsupported op, etc), unless
+  //    it’s considered fatal to the device (e.g. might crash the whole device
+  //    routing daemon).
```

Comment:
Are there any such cases that would crash the daemon that we can know about? I feel like NPE might be one, but this is something that we could pre-check for (i.e., check that the operation != nil - if it does, return an error directly).

Reply:
I think this is a good point. Changed this point to use stronger word. Please help take a look again. Thanks.